### ISSUE TITLE
Update the Upgrading Guide regarding V23

### ISSUE THREAD

**Issue by vursen** [vaadin/docs#951]
Here is the list of PRs with changes targeting V23 that should be mentioned in the Upgrading Guide once the version is released.

## Breaking changes

- [ ] https://github.com/vaadin/web-components/pull/3006
- [ ] https://github.com/vaadin/web-components/pull/3032
- [ ] https://github.com/vaadin/web-components/pull/3106
- [ ] https://github.com/vaadin/web-components/pull/3108
- [ ] https://github.com/vaadin/web-components/pull/3109
- [ ] https://github.com/vaadin/web-components/pull/3135
- [ ] https://github.com/vaadin/web-components/pull/3146
- [ ] https://github.com/vaadin/web-components/pull/3167

## Accessibility changes

- [ ] https://github.com/vaadin/web-components/pull/3057
- [ ] https://github.com/vaadin/web-components/pull/3088
- [ ] https://github.com/vaadin/web-components/pull/3107
- [ ] https://github.com/vaadin/web-components/pull/3156
- [ ] https://github.com/vaadin/web-components/pull/2395
- [ ] https://github.com/vaadin/web-components/pull/3168
- [ ] https://github.com/vaadin/web-components/pull/3192
- [ ] https://github.com/vaadin/web-components/pull/3221

## Polymer API removal

- [ ] https://github.com/vaadin/web-components/pull/3092
- [ ] https://github.com/vaadin/web-components/pull/3150
- [ ] https://github.com/vaadin/web-components/pull/3195
- [ ] https://github.com/vaadin/web-components/pull/3196
- [ ] https://github.com/vaadin/web-components/pull/3197
- [ ] https://github.com/vaadin/web-components/pull/3200
- [ ] https://github.com/vaadin/web-components/pull/3201
- [ ] https://github.com/vaadin/web-components/pull/3205
- [ ] https://github.com/vaadin/web-components/pull/3206
- [ ] https://github.com/vaadin/web-components/pull/3211
- [ ] https://github.com/vaadin/web-components/pull/3213
- [ ] https://github.com/vaadin/web-components/pull/3207

## Other changes

- [ ] https://github.com/vaadin/web-components/pull/2784
- [ ] https://github.com/vaadin/web-components/pull/2990
- [ ] https://github.com/vaadin/web-components/pull/3025
- [ ] https://github.com/vaadin/web-components/pull/3031
- [ ] https://github.com/vaadin/web-components/pull/3085
- [ ] https://github.com/vaadin/web-components/pull/3093
- [ ] https://github.com/vaadin/web-components/pull/3156
- [ ] https://github.com/vaadin/web-components/pull/3164

*The list is not complete. Feel free to add your PR if it targets V23.*

**Comment by jouni** [comment]
I assume these were covered in the V14 → V23 upgrade guide.
